WikipediaLibrary/TWLight
# Add searchable field to partners and display icon on searchability status

[//]: # (Thank you for uploading a PR to the Wikipedia Library!)

## Description
Added the `searchable` field to the Partner model and added a search icon that displays on collection tiles whether it is searchable via EDS or not.

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
EDS only indexes a subset of our total available content. Users might want to make informed decisions about whether they should search in EDS or at the publisher's website. We can help with this by denoting in some way which resources are indexed.

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T270115](https://phabricator.wikimedia.org/T270115)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)
1. Run the migrations (if you haven't)
2. Modify some collections in the `admin` interface so they are searchable and partially searchable.
3. Navigate to My Library
4. Search icons should appear on the collections that are searchable or partially searchable

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)

![Screen Shot 2021-09-14 at 17 53 17](https://user-images.githubusercontent.com/7854953/133347999-309a3f07-123e-4398-89db-4a7809fef4d5.png)

![Screen Shot 2021-09-14 at 17 53 35](https://user-images.githubusercontent.com/7854953/133348003-fa5b19c5-5f37-45c0-823c-7622a8bdce98.png)

![Screen Shot 2021-09-17 at 17 41 50](https://user-images.githubusercontent.com/7854953/133860930-08d68973-62cb-4335-97bd-5c5b0b3ed890.png)


## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
